homeworks/homework_24/main.js

```diff
@@ -0,0 +1,89 @@
+/* Работа с переменными
+важность: 2
+1.	Объявите две переменные: admin и name.
+2.	Запишите строку "Джон" в переменную name.
+3.	Скопируйте значение из переменной name в admin.
+4.	Выведите на экран значение admin, используя функцию alert (должна показать «Джон»). */
+
+let admin, name;
+name = "Джон";
+admin = name;
+alert( admin);
+
+/* Исправьте сложение
+Ниже приведён код, который запрашивает у пользователя два числа и показывает их сумму.
+Он работает неправильно. Код в примере выводит 12 (для значения полей по умолчанию).
+В чём ошибка? Исправьте её. Результат должен быть 3.  */
+
+let a = +prompt("Первое число?", 1);
+let b = +prompt("Второе число?", 2);
+alert(a + b);
+
+// 1. Запросите у пользователя его имя и выведите в ответ: «Привет, его имя!».
+
+let name = prompt("what is your name?", "");
+alert(`Hello, ${name}`);
+
+// 2. Запросите у пользователя год его рождения, посчитайте, сколько ему лет и выведите результат. Текущий год укажите в коде как константу.
+
+let birthYear = prompt("what is your birth year?", "");
+const currentYear = 2020;
+let age = currentYear - birthYear;
+alert( `Your age is ${age}` );
+
+// 3. Запросите у пользователя длину стороны квадрата и выведите периметр такого квадрата.
+
+let squareSide = prompt("lenght of one side of a square", "");
+let squareArea = squareSide ** 2;
+alert(squareArea);
+
+// 4. Запросите у пользователя радиус окружности и выведите площадь такой окружности.
+
+let circleRadius = prompt("radius of a circle", "");
+const Pi = 3.14159265359;
+let circleArea = Pi * circleRadius ** 2;
+alert(circleArea);
+
+// 5. Запросите у пользователя расстояние в км между двумя городами и за сколько часов он хочет добраться. Посчитайте скорость, с которой необходимо двигаться, чтобы успеть вовремя.
+ let distance = prompt("distance between cities", "");
```

Comment:
= prompt("distance between cities", ""); if you do not want to pass default value than you can write only prompt("distance between cities")